treesapp_assign_and_update.Rmd

```diff
@@ -87,41 +99,81 @@ treesapp update \
+Protein families vary in their evolutionary complexity; many, especially genes involved in scavenging and metabolizing nutrients, have often been involved in some gene duplication and/or lateral gene transfer event while others have remained sessile within their host. Moreover, closely related enzymes can vary in their activities and it is often valuable, though not necessary, to annotate such features.
+
+If you are not familiar with the evolution of a protein family it may be helpful to find reference sequences for the activity, function, or other feature you're interested in and place them on the tree with `treesapp assign`. Then, guided by these sequences, it should be easier to colour the clades. As an example, let's place some sequences of ammonia-oxidizing Bacteria (AOB) on the phylogeny.
+
 ```{bash eval = FALSE}
 treesapp assign \
   -n 4 -m prot --trim_align \
   --refpkg_dir XmoA_FunGene_update/final_outputs/ \
   --fastx_input uniprot-AmoA.fasta \
   --output AmoA_assign/
 ```
+
 Creating your own annotations involves scouring the literature to identify the phenotypes associated with each organism or taxon.For this tutorial you can use the annotations already created for the XmoA reference package.
+
 ```{bash eval = FALSE}
 wget https://raw.githubusercontent.com/hallamlab/RefPkgs/master/Nitrogen_metabolism/Nitrification/XmoA/XmoA_taxonomy-phenotype_map.tsv
 ```
-To create a file compatible with iTOL you will need to convert the taxonomy-phenotype mapping file with treesapp colour, 
+
+To create a file compatible with iTOL you will need to convert the taxonomy-phenotype mapping file with `treesapp colour`. 
+
 ```{bash eval = FALSE}
 treesapp colour \
   -r XmoA_FunGene_update/final_outputs/XmoA_build.pkl \
   -o XmoA_colours/ \
   -n Function \
   --taxa_map XmoA_taxonomy-phenotype_map.tsv
 ```
+
 The two output files -- `XmoA_Function_colours_style.txt` and `XmoA_Function_colour_strip.txt` -- can be used to bring colour to the JPlace file in iTOL and also be used for adding phenotype-level annotations to the classification table with `treesapp layer`.Let's make copies of the two text files in  XmoA_colours/ into the AmoA_assign/iTOL output directory  and transfer them to desktop.
+
 ```{bash eval = FALSE}
 cp -r XmoA_colours   AmoA_assign/iTOL_output/
 scp -r  root@10.19.139.201:/data/ts_tutor/AmoA_assign/iTOL_output  < path_to_your_folder> 
 ```
```

Comment:
````suggestion
```{bash eval = FALSE}
cp -r XmoA_colours AmoA_assign/iTOL_output/
scp -r root@10.19.139.201:/data/ts_tutor/AmoA_assign/iTOL_output <path to Xmoa_sequences>
```
````